src/optimisation/linear-problem-api/include/antares/optimisation/linear-problem-api/ILinearProblemData.h

```diff
@@ -2,17 +2,30 @@
 #pragma once
 
 #include <string>
+#include <vector>
 
 namespace Antares::Optimisation::LinearProblemApi
 {
 
+struct DataSeriesKeys
```

Comment:
what is the purpose of this struct ?

Reply:
i was using it to "transport"  scenario and scenarioGroup from main to where they were supposed to be used.
Since scenario is not managed in this pr, i will have to update